routes/api/familyRouter.js

```diff
@@ -22,4 +24,9 @@ router.delete("/deleteFamilyMember/:name", deleteFamilyMember);
 
 // localhost:3000/api/family/updateFam/:name
 router.put("/updateFam/:name", updateFamilyMember)
-module.exports = router
\ No newline at end of file
+
+router.get("/members", getAllFamily);
+router.post("/members", createOneFamilyMember);
+router.get("/members/:id", getMemberById);
+router.patch("/members/:id", patchFamilyMember);
```

Comment:
This is a very common pattern for handling CRUD operations. A resource (in this case, `members`) is used as the name of an endpoint. Operations that do not target a preexisting user (search/get all/create a new one) operate off of `/:resource` where as operations that target a particular existing resource instance (put/patch/delete) operate off of `/:resource/:id`

Note the use of `patch` instead of `put` here. Semantically, a `patch` operation implies updating _only the specified parts of a given record._